### PR TITLE
Update diffusers default snippets to be complete.

### PR DESCRIPTION
**Pull request by Vaibhavs10**
Slack chat for ref: https://huggingface.slack.com/archives/C04PJ0H35UM/p1726563327331199?thread_ts=1725899853.249099&cid=C04PJ0H35UM

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]

**Review by Wauplin — COMMENTED** [review]
(expect style linting)

**Comment by mishig25** [comment]
the failing tests are unrelated to this pr

**Comment by Vaibhavs10** [comment]
Will wait for @sayakpaul or @apolinario to approve before merging!

Thanks @mishig25 for fixing the lint issues 🐐

File: packages/tasks/src/model-libraries-snippets.ts
Comment (gary149) [inline packages/tasks/src/model-libraries-snippets.ts:137]: Maybe a simpler prompt will work better as it will be applied on all type diffusion models? (cc @apolinario)

```suggestion
prompt = "Astronaut in a jungle"
```

Comment (sayakpaul) [reply packages/tasks/src/model-libraries-snippets.ts:137]: Adding those few keywords just exceptionally boost the quality.

**Review by pcuenca — APPROVED** [review]
Looks good!

File: packages/tasks/src/model-libraries-snippets.ts
Comment (gary149) [reply packages/tasks/src/model-libraries-snippets.ts:137]: Ok then!

File: packages/tasks/src/model-libraries-snippets.ts
Comment (julien-c) [inline packages/tasks/src/model-libraries-snippets.ts:135]:
```suggestion
pipe = DiffusionPipeline.from_pretrained("${model.id}")
```
like in transformers? (unsure, up to you, no big difference lol)

Comment (sayakpaul) [reply packages/tasks/src/model-libraries-snippets.ts:135]: Not a big deal I am okay with `pipe`, too.

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries-snippets.ts:135]: `pipe` looks good! I'll update it in all the other snippets too.

File: packages/tasks/src/model-libraries-snippets.ts
Comment (osanseviero) [inline packages/tasks/src/model-libraries-snippets.ts:138]: Nit: as discussed in Slack, having a prompt might be a bit confusing due to many models expecting certain tokens. In any case, this is fine I think to showcase usage

**Review by osanseviero — APPROVED** [review]

**Review by Vaibhavs10 — COMMENTED** [review]
For LoRA models would it make sense to read the examples from the README? or is that an overkill?

ref: https://huggingface.co/Shakker-Labs/FLUX.1-dev-LoRA-AntiBlur/blob/main/README.md?code=true#L12

**Comment by apolinario** [comment]
Well constructed LoRA models have an `instance_prompt` variable which could be used in the example prompt, but I like the idea of taking the first `gallery` prompt if existing. IMO not overkill and would make the UX great  https://huggingface.co/multimodalart/vintage-ads-flux/blob/main/README.md?code=true#L9

Otherwise lgtm

**Review by apolinario — APPROVED** [review]

**Comment by Vaibhavs10** [comment]
Will follow up w/ showing Gallery prompts in a subsequent PR. Thanks for the reviews everyone!

**Pull request merged**
